lib/matplotlib/axes/_axes.py

```diff
@@ -11,6 +11,7 @@
 from numpy import ma
 
 import matplotlib
+from matplotlib import unpack_labeled_data
```

Comment:
Could you please file a bug report, since this feature has already been
merged?

On Sun, Mar 20, 2016 at 4:37 PM, mihitha nutakki <notifications@github.com>
wrote:

> In lib/matplotlib/axes/_axes.py
> <https://github.com/matplotlib/matplotlib/pull/5053#discussion_r56771530>:
>
> > @@ -11,6 +11,7 @@
> >  from numpy import ma
> >
> >  import matplotlib
> > +from matplotlib import unpack_labeled_data
>
> This is not working
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/5053/files/367b68fbb7added330160e426830116cd247212a..34b879abb0b1c2976dac630d77289020e81494e5#r56771530>
>